examples/bri-3/docs/nats/nats-configuration.md

```diff
@@ -0,0 +1,59 @@
+# NATS configuration
+
+We are relying on simple user authorization. Each BpiSubject is represented with a single user in Nats. Each Nats user is authorized for specific topics through the permissions map in nat server congif file. Currenly, we assume that the Bpi Admin will manage these users and their permssions. [Link](    https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro) to Nats docs.
+
+NATS config that can be used in the development environment can be found in here *./example-nats-server.conf*
+
+[More](https://github.com/nats-io/nats-server/tree/main/server/configs) NATS sample configurations.
+
+## Authorization flow
+
+1) On server initialization, BPI operator will create a main BPI user with a password and add it to server configuration. Example can be found in *./example-nats-server.conf* file, bpi_operator user.
```

Comment:
how would this conf file be stored/backuped in real life scenario?

Reply:
Up to the admin. You would probably use a docker volume with regular daily backups.